reactor-test/src/test/java/reactor/test/StepVerifierTests.java

```diff
@@ -1114,7 +1116,8 @@ public void verifyVirtualTimeOnErrorAsync() {
 		            .verify();
 	}
 
-	@Test(timeout = 1000)
+	@Test
+@Timeout(value = 1000, unit = TimeUnit.MILLISECONDS)
```

Comment:
the indentation seems to be wrong:
<img width="557" alt="Screenshot 2020-09-07 at 5 04 57 PM" src="https://user-images.githubusercontent.com/1050762/92400474-4794a380-f12c-11ea-9a65-73b2ea7cd9e6.png">

mixed tabs/spaces?

Reply:
Will fix and propagate same kind of changes to 3.3.x and master (which may have more tests).

Thanks for the review!